src/completion_queue.rs

```diff
@@ -205,7 +206,7 @@ impl From<WCError> for io::Error {
 }
 
 /// Work request id
-#[derive(PartialEq, Eq, Hash, Debug, Clone, Copy)]
+#[derive(PartialEq, Eq, Hash, Debug, Clone, Copy, Serialize, Deserialize)]
```

Comment:
Why do you add these traits?

Reply:
This was added to enable WorkRequestID to replace RequestID of Agent. But we'll refactor this part to make it more logical to abstract.